app/src/main/java/org/greenstand/android/TreeTracker/api/models/requests/DeviceRequest.kt

```diff
@@ -23,4 +24,7 @@ data class DeviceRequest(@SerializedName("app_version")
                          @SerializedName("androidRelease")
                          val androidRelease: String = Build.VERSION.RELEASE,
                          @SerializedName("androidSdkVersion")
-                         val androidSdkVersion: Int = Build.VERSION.SDK_INT)
\ No newline at end of file
+                         val androidSdkVersion: Int = Build.VERSION.SDK_INT,
+                         @SerializedName("instance_id")
+                         val instance_id: String = FirebaseInstanceId.getInstance().id
```

Comment:
Great job! Can we just make sure this field is camelCase like the rest?

Reply:
Yes! Just changed it!